games-arcade/vvvvvv_data/vvvvvv_data-2.0.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Data for VVVVVV"
+DESCRIPTION="Notice requested by the author: VVVVVV is a commercial game! The author has \
+given special permission to make this Haiku version available for free. If you \
+enjoy the game, please consider purchasing a copy at thelettervsixtim.es. \
+
+Data file from the Make and Play Edition of VVVVVV. Distributed with permission from \
+Terry Cavanagh (game creator) as a license exception. More info available here: \
+https://github.com/TerryCavanagh/VVVVVV/blob/master/License%20exceptions.md
+HOMEPAGE="https://thelettervsixtim.es"
+COPYRIGHT="Terry Cavanagh 2010-2020"
+LICENSE="VVVVVV Source Code License v1.0"
+REVISION="1"
+SOURCE_URI="https://thelettervsixtim.es/makeandplay/data.zip"
+CHECKSUM_SHA256="6fae3cdec06062d05827d4181c438153f3ea3900437a44db73bcd29799fe57e0"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	vvvvvv_data = $portVersion >= 0
+	"
+REQUIRES="
+	vvvvvv = $portVersion compat >= 0
```

Comment:
Why add this requirement?

Reply:
This came from the `moleinvasion_music` recipe which was used as a base, following the suggestion of @Begasus.